PIL/Image.py

```diff
@@ -570,18 +603,21 @@ def close(self):
 
     if sys.version_info >= (3, 4, 0):
         def __del__(self):
+            # type: () -> None
             if (hasattr(self, 'fp') and hasattr(self, '_exclusive_fp')
                and self.fp and self._exclusive_fp):
```

Comment:
Again, this might be a case of using `ignore` for now and adding an issue to do some type-inspired tidying ;)